docs/New-Maintainer-Checklist.md

```diff
@@ -25,12 +28,15 @@ A few requests:
   button in "create a merge commit" mode for Homebrew/brew or "squash and
   merge" for a single formulae change. If in doubt, check with e.g. GitX that
   you've not accidentally added merge commits
-- still create your branches on your fork rather than in the main repository
-- if still in doubt please ask for help and we'll help you out - these are
-  probably worth a read:
-    - https://github.com/Homebrew/brew/blob/master/docs/Brew-Test-Bot-For-Core-Contributors.md
-    - https://github.com/Homebrew/brew/blob/master/docs/Maintainer-Guidelines.md
-    - possibly everything else in the documentation
+- still create your branches on your fork rather than in the main repository.
+  Note GitHub's UI will create edits and reverts on the main repository if you
+  make edits or click revert on the Homebrew/brew repository rather than your
+  own fork.
```

Comment:
I reckon use GUI for reverts and not edits ideally but I actually don't care that much about either if they are quick. Thoughts on rewording as a result?